pkg/controller/installation/products/codeready/reconciler_test.go

```diff
@@ -189,7 +191,7 @@ func TestCodeready_reconcileCluster(t *testing.T) {
 	}{
 		{
 			Name:           "test phase in progress when che cluster is missing",
-			ExpectedStatus: v1alpha1.PhaseInProgress,
+			ExpectedStatus: v1alpha1.PhaseAwaitingComponents,
```

Comment:
We could potentially also add another test case where the `postgres` cr contains a complete status, and the cro generated connection secret already exists in the mock cluster, so the expected phase is `PhaseInProgress`

Reply:
hadn't much joy with that so will leave it for now.